recipe/meta.yaml

```diff
@@ -11,7 +11,6 @@ source:
 
 build:
   number: 0
-  skip: true  # [py3k]
```

Comment:
You need to also remove the conda_build_config.yaml to get py3 builds.

Reply:
Besides enabling py2, is there any reason to keep the conda_build_config.yaml?

Reply:
Nope.